### ISSUE TITLE
Cameras issue imprting from Zephyr to Meshlab

### ISSUE THREAD

**Issue by alshortyq** [cnr-isti-vclab/meshlab#776]
#### Check all platforms you experienced the issue (change to `[x]`)
- [x ] Windows
- [ ] Mac OS
- [ ] Linux
- [ ] Every platform

#### What type of issue is?
- [ x] bug or unexpected behavior
- [ ] crash
- [ ] feature request
- [ ] other (specify...)

#### Describe your issue
Hi!

When i try to import cameras from Zephyr aerial to Meshlab useing Boundle format i got all correct but all the cameras turned AWAY from target (not TO target). Please see the screenshots:

from Zephyr: https://ibb.co/sgs5kC0
from Meshlab: https://ibb.co/31Q25sw

3Dflow Staff told me that "I'm afraid that's an import issue from Meshlab, as bundler export seems to be working correcly on Zephyr's end."

Tnx.
...

####Guidelines for a good issue

Before submitting an issue: 

- be as detailed as possible. If the issue is related to a specific mesh or file, please attach it. Make sure that the problem is reproducible; 
- specify the operative system you are using and its version/distribution; if it is possible, verify if the problem persists in other platforms;
- bspecify the MeshLab version you are using and be sure that your issue also happens on the last MeshLab version;
- check if your issue has been already reported by someone else. In this case, do not open another issue: just add your experience in the existing issue;
- If the issue is specific of a platform, please specify it on the subject in square brackets (example: `[MacOS] crash when opening obj`);
- delete these guidelines before submitting :)

**Comment by alshortyq** [comment]
and if Zephyr works fine then there is one more issue: there are quotes in filenames of images in the cameras list, but Meshlab does not load these images correctly until i delete those quotes manually in filenames of images.

**Comment by alshortyq** [comment]
XMP import from Zephyr Aerial 4.53 also does not work 

- first: in Meshlab it calls XPM (not XMP), but i renamed the files manually
- second: there is no error during import but also no cameras imported
- third: i see raster image in the list of Neshlab panel, but with the message "Image file C:/Work/Alex/IMG_1990.xpm has not been correctly loaded, a fake image is going to be shown."

**Comment by alemuntoni** [comment]
The extension is actually XPM: https://it.wikipedia.org/wiki/XPM
I have to check for the camera and quotes issues. Could you please share bundler files having these problems?

**Comment by alshortyq** [comment]
Sure: Here are the files with full information 

https://dropmefiles.com/8sBpP

clear_zephyr_export.zip - is the clear zephyr aerial 4.53 export cameras in Bundle format. To load it to Meshlab 2020.03 you yave to open project and select .out - file. Then you will be prompted to select the cameras list. You have to choose "externals.txt"  If there are quotes in filenames of images - then Meshlab does not load them properly. (i sent you the files without actual images  because of a big amount of data. You can emulate the presence of the images by using your images). And  at the end - you will get cameras turned AWAY from target.

It is strange, but if i use Bundle export from Agisoft PhotoScan Professional and use externals.txt file from Zhephyr export (some kind of shamanism) - all cameras turned TO the target (but i think they all have wrong absolute position because of they are from the different SFM-positioning). Bundle file from Agisoft PhotoScan Professional export you can see as well.

So i think the "quotes" problem - it is the first problem (but it can be easily fixed by removing all the quotes manually), but the problem with the wrong cameras orientation - is the MAIN problem - and it is there - where Meshlab reads Bundle export from Zephyr... 

Coud you please make an option to load the export from Zephyr?

**Comment by alshortyq** [comment]
Hi! Is there any news about correct camera orientation during Bundle export from Zephyr?

**Comment by alemuntoni** [comment]
Nope, sorry @alshortyq. We plan to make a major revision for the part of MeshLab that loads and saves project files, but I cannot say when right now.

**Comment by alshortyq** [comment]
I see, but is there any way to import cameras to Meshlab except importing from VisualSFM?  (Import from VisualSFM works correctly, but it is very old programm - for example, it loses some data...)

tnx.

**Comment by ttsesm** [comment]
Related to this issue, is it possible to load Meshroom .sfm camera poses and raster images in Meshlab?

**Comment by alemuntoni** [comment]
@ttsesm nope, sfm file format is not supported by meshlab. I actually can't find a proper documentation describing the file format...

**Comment by ttsesm** [comment]
Yes, it seems that is not that well documented. I do not know if this [link](https://github.com/alicevision/meshroom/wiki/Using-known-camera-positions) could help somehow to get a better insight.
